libcxx/include/module.modulemap

```diff
@@ -200,6 +200,10 @@ module std_core [system] {
       header "__type_traits/is_fundamental.h"
       export std_core.type_traits.integral_constant
     }
+    module is_implicit_lifetime {
+      header "__type_traits/is_implicit_lifetime.h"
+      export std_core.type_traits.is_implicit_lifetime
```

Comment:
You don't need to export the module itself. Did you mean `std_core.type_traits.integral_constant`?

Reply:
Thanks!